### PR TITLE
Fix duplicate command output on SSHKit 1.7.1

### PR DESCRIPTION
**Pull request by robd**
Unfortunately this bug is untestable with our current acceptance tests due to historical differences between the Local and SSHKit backends. See #42 for more details.

### CODE REVIEW

File: lib/airbrussh/delegating_formatter.rb [hunk 1]
```diff
@@ -26,11 +26,18 @@ def initialize(formatters)
       end
     end
 
-    # SSHKit's formatters mutate the stdout and stderr data in the
-    # command obj. So we need to dup it to ensure our copy is unscathed.
+    # For versions of SSHKit prior to 1.7.1, the LogfileFormatter and
```
Comment (mattbrictson) [inline lib/airbrussh/delegating_formatter.rb]: I think this should be "For SSHKit versions 1.7.1 and prior".

**Comment by mattbrictson** [comment]
I verified the fix using a capistrano deploy as well. Thanks!

**Comment by robd** [comment]
OK, I made the comments a bit clearer and fixed the SSHKit 1.7.1 inaccuracy.

**Comment by mattbrictson** [comment]
:+1:

**Pull request merged**